src/test/util/test_file.rs

```diff
@@ -0,0 +1,55 @@
+use crate::test::util::test_event::TestEvent;
```

Comment:
I think the test runner files introduced in this PR fit better somewhere like `src/test/spec/runner`, since they're specifically for spec tests, and they're also more than just extra utilities.